src/components/Footer.js

```diff
@@ -83,7 +83,7 @@ export const Footer = () => {
           <GoMarkGithub size={20} /> &nbsp; @smswithoutborders
         </a>
       </div>
-      <div className="mb-40 w-56">
+      <div className="mb-10 w-56">
```

Comment:
The footer height was increased so that the Language switcher won't be hidden on mobile screens. Try changing the language with this height to see how it behaves.  You may need to try another approach

Reply:
I figured that was why you put it, although it expands the footer on click . Have you tried it?